### DISCUSSION TITLE
[M-Z] (2024/25) - sez. 0.1 - es. 1.6 (30 settembre)

### DISCUSSION THREAD

**Discussion by marcoseverini** [sapienzastudentsnetwork/calcolo-delle-probabilita#280] [Esercizi M-Z]
<img width="523" alt="immagine" src="https://github.com/user-attachments/assets/7d8bf598-a07a-4847-bf62-b4e1b3495455">

**Comment by marcoseverini** [comment]
<img width="713" alt="immagine" src="https://github.com/user-attachments/assets/8301ba24-a967-4621-8a1a-abc1b97c3b8c">

**Comment by scuty2000** [comment]
Un modo diverso di interpretare il problema è quello di scegliere prima i 4 bambini a cui dare i libri, e poi calcolare i modi in cui questi possono essere distribuiti. Quindi:

![image](https://github.com/user-attachments/assets/c35f38ed-728a-4999-a19a-36edae467b43)
